payload/Library/Application Support/umad/Resources/umad

```diff
@@ -750,16 +743,15 @@ def main():
         umad.views['field.depfailuresubtext'].setHidden_(False)
         umad.views['field.depfailuresubtext'].setStringValue_(
             manualenroll_h2_text)
-    else:
-        if date_diff_seconds <= 0:
-            umad.views['field.manualenrollmenttext'].setHidden_(True)
-            umad.views['image.nagscreen'].setHidden_(False)
-            umad.views['field.depfailuretext'].setHidden_(False)
-            umad.views['field.depfailuretext'].setStringValue_(
-                opts.depfailuretext)
-            umad.views['field.depfailuresubtext'].setHidden_(False)
-            umad.views['field.depfailuresubtext'].setStringValue_(
-                opts.depfailuresubtext)
+    elif date_diff_seconds <= 0:
```

Comment:
Only thing I disagree with. Please keep as is with the else and an if. In this case they are functionality different.

Reply:
However you want it is totally fine. Either way works exactly the same... unless you are planning on adding stuff outside of the `if_date_diff_seconds <= 0:` block (i.e. right before the `check_mdm_status(True)`.

Reply:
I think I did and then stopped. You're right. Let's keep it.